source/Frame.lua

```diff
@@ -78,12 +78,15 @@ local function Frame(name, parent)
         return false
     end
 
+    ---@class Frame
     object = {
         barActive = false,
         barBackground = colors.gray,
         barTextcolor = colors.black,
         barText = "New Frame",
         barTextAlign = "left",
+        isMovable = false,
+        ---@deprecated deprecated in favor of Frame#isMovable
```

Comment:
added `@deprecated` notice for emmy lua users, provides helpful comments when typing the function